### ISSUE TITLE
Players are not always asked which order to resolve their triggers in

### ISSUE THREAD

**Issue by ais523** [mtgred/netrunner#5861]
Link to bug replay: https://jinteki.net/bug-report/9bae9250-767e-4b56-a3ae-5b2baeb1de6e?b=0

Description:

jinteki.net seems inconsistent in whether it asks which order to resolve triggers in. In this game, on the last click of the game, the Runner stole two copies of Sting!; that creates three triggers (the Runner's Pantograph trigger, the Corp's Sting! trigger, and the Corp's Jinteki: Personal Evolution trigger). Neither steal asked the Corp which order to resolve the triggers in (even though it normally does ask even when both triggers do 1 damage, and it could matter if, e.g., the Runner has I've Had Worse in hand). Expected behaviour is for the Pantograph trigger to resolve first, and then the Corp to choose which of the Corp's triggers to resolve first.

I'm not sure whether the Pantograph trigger caused this bug to happen, or whether it was irrelevant and the cause is something else (e.g. a buggy attempt to not ask when there are multiple identical triggers); as a consequence, I've had to leave the title fairly vague.

**Comment by NoahTheDuke** [comment]
You're right that it's inconsistent. That's by design. We don't want to ask for input when you're taking credits off 4 different cards, what does it matter which order you choose? So we do it piecemeal as places where it's desired are found.

We can easily add it where you've mentioned. Thanks for letting us know!

**Comment by NBKelly** [comment]
Closed as completed (PE is now always interactive)
https://github.com/mtgred/netrunner/blob/6e9e6c1503bce13b2deb4584588f89e01fa52c35/src/clj/game/cards/identities.clj#L1261
